ingest-find-existing-asset/src/test/scala/uk/gov/nationalarchives/ingestfindexistingasset/testUtils/ExternalServicesTestUtils.scala

```diff
@@ -175,6 +175,8 @@ class ExternalServicesTestUtils extends AnyFlatSpec with BeforeAndAfterEach with
       val serveEvents = dynamoServer.getAllServeEvents.asScala.toList
 
       serveEvents.size should equal(numOfDynamoGetRequests + numOfDynamoUpdateRequests)
+      val d = serveEvents.map(_.getRequest.getBodyAsString)
```

Comment:
I'm guessing these two lines were left in by mistake?

Reply:
Ah yeah, I was debugging. I've removed them.